### DISCUSSION TITLE
Naviguer a travers plusieurs Synapps depuis un seul wit

### DISCUSSION THREAD

**Discussion by ABINolan** [witsa/synapps#766] [Q&A] [answered]
Bonjour à tous,

Ayant plusieurs wit a installer sur un réseaux local je souhaite naviguer a tyravers plusieurs Synapps depuis 1 seul wit maître.
Cependant je ne connais plus la commande qu'il faut renseigner pour pointer sur la scène de l’autre synapps.
Voici ce que j'ai essayé :

![Uploading Synapps.PNG…]()

En vous remerciant,

Nolan

**Comment by JeremieDENAEGEL** [comment]
Bonjour,

On ne voit pas l'image
Du coup, tu veux afficher une autre synapps, présente dans le redy principale ou sur un autre redy, dans un acteur écran ou qu'elle s'affiche sur toute la page ?

**Comment by Synfactory** [comment, marked as answer]
Salut,
peut-être la réponse dans ce post?
 #610
